index.html

```diff
@@ -114,6 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Serhii Murashko</p>
+          <p class="contact">
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/simplesquirrel/" target="_blank">simplesquirrel</a>
+          </p>
+          <p class="about">Future Automation QA Engineer</p> 
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="youtube.com/" target="_blank" title="First Resource">
+                  1000 and 1 course for you
+                </a>
+              </li>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="Second Resource">
+                  Only one who is going to listen to your problems
+                </a>
+              </li>
+              <li>
+                <a href="https://learngitbranching.js.org" target="_blank" title="Third resource">
+                  Learn git by playing game
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this `*END TEMPLATE*` part into your name as well. It would be easier for everyone to identify your card in code.